flex/engines/http_server/handler/hqps_http_handler.h

```diff
@@ -78,6 +91,14 @@ class hqps_adhoc_query_handler : public seastar::httpd::handler_base {
   std::vector<executor_ref> executor_refs_;
   std::vector<codegen_actor_ref> codegen_actor_refs_;
   bool is_cancelled_;
+#ifdef HAVE_OPENTELEMETRY_CPP
+  opentelemetry::nostd::unique_ptr<opentelemetry::metrics::Counter<uint64_t>>
+      total_counter_;
+  opentelemetry::nostd::unique_ptr<opentelemetry::metrics::Counter<uint64_t>>
+      success_counter_;
```

Comment:
Use a total_counter and append label to denote success. This would reduce the amount of data to export.